sympy/physics/mechanics/particle.py

```diff
@@ -124,7 +124,7 @@ def angular_momentum(self, point, frame):
         The angular momentum H, about some point O of a particle, P, is given
         by:
```

Comment:
Maybe another improvement for the sake of consistency is either have a `:` after every `given by` or remove it everywhere.